dune-project

```diff
@@ -21,5 +21,6 @@
   logs
   containers
   fmt
-  (cmdliner (>= 1.1.0))
-  (ocamlformat (and :with-dev-setup (= 0.26.2)))))
```

Comment:
I think you removed this line by mistake.

Reply:
Hello sir @NathanReb ,

Thank you for the clarification. I’ve made the necessary recommendations and implemented the cram test. Please take a look to see if there’s still room for improvement.

Thank you!